walk/git.go

```diff
@@ -38,7 +38,7 @@ func (g *GitReader) Read(ctx context.Context, files []*File) (n int, err error)
 		r, w := io.Pipe()
 
 		// create a command which will execute from the specified sub path within root
-		cmd := exec.Command("git", "ls-files")
+		cmd := exec.Command("git", "ls-files", "--cached", "--others")
```

Comment:
Probably want to add `--exclude-standard` so it respects gitignore.

Reply:
Thanks https://github.com/numtide/treefmt/pull/565